docs/source/tutorial.rst

```diff
@@ -210,7 +210,7 @@ Workflows
 =========
 
 
-Implementing a simple workflow
+Implementing a simple workflow with signac-flow
```

Comment:
```suggestion
Implementing a simple workflow
```
I think we should keep this as it is and change the current heading **Workflows** with **Workflows with signac-flow**

Reply:
I would do neither. I believe the correct place to mention `signac-flow` is in line 217. Something like,

````
For this, we will incoporate the ``compute_volume()`` function into a workflow using the package ``signac-flow`` and its :class:`~.flow.FlowProject` class.
````

Reply:
this is fine by me